website/static/js/pages/project-base-page.js

```diff
@@ -27,35 +27,33 @@ $(window).unload(function(){
     return 'Unload';
 });
 
-// TODO: See if this can be removed
 $(document).ready(function() {
-    $.getJSON(node.urls.api, function(data) {    
+    $.getJSON(node.urls.api, function(data) {
         $('body').trigger('nodeLoad', data);
     });
+});
+
+$(window).scroll(function() {
+    var st = $(this).scrollTop();
```

Comment:
Minor detail, but `st` is not a very descriptive variable name. Can you call it `scrollTop()` instead?